torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11396,6 +11406,30 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
            error_inputs_func=error_inputs_max_pool2d,
            sample_inputs_func=sample_inputs_max_pool),
+    OpInfo('max_pool2d_with_indices_backward',
+           aten_name=None,
```

Comment:
Doesn't this have an aten name?

Reply:
I added a custom op because we also need to run the forward to get the grad_out tensor (need the out shape to create grad_out). so, hence no corresponding aten name